ipn/ipnlocal/cert.go

```diff
@@ -540,6 +585,20 @@ func acmeKey(cs certStore) (crypto.Signer, error) {
 	return privKey, nil
 }
 
+func acmeClient(cs certStore) (*acme.Client, error) {
```

Comment:
Style pico-nit: Personally I like for function names to always be verb phrases (e.g. `getACMEClient`), but that might just be me. Either way is fine.

Reply:
I always thought that the Go style was generally to avoid verbs unless they provide more clarity, e.g. https://pkg.go.dev/net/http#Request.Cookie vs `GetCookie`.
But this may just be my Java scars, with all those getters and setters :scream: